### ISSUE TITLE
dose the chain rely on  njobs ?

### ISSUE THREAD

**Issue by dreamqin68** [pymc-devs/pymc#2213]
The sample.py write that:
 chain : int
        Chain number used to store sample in backend. If `njobs` is
        greater than one, chain numbers will start here.
    njobs : int
        Number of parallel jobs to start. If None, set to number of cpus
        in the system - 2.
when I write:
  trace = pm.sample(n_samples,step=step,chain=2) in my code, the trace.nchains=1
  why?

**Comment by twiecki** [comment]
what do you want to achieve? multiple chains without running them in parallel?

**Comment by dreamqin68** [comment]
@twiecki  No. my cpus=16 ,so the njobs=14 in default. I want to have a right answer of gelman-rubin.

**Comment by twiecki** [comment]
what happens if you not set chain=2 but just `njobs=14`?

**Comment by ColCarroll** [comment]
tl;dr: I agree with @twiecki 

Setting `chain` yourself isn't very common -- if `njobs=14`, then by default you will have an array of 14 chains after sampling (each with the number of steps requested).  It looks like you could then sample again, passing in the existing trace, and setting `chain=7, njobs=7` (for example), and new samples will be concatenated to chains 7-14.

**Comment by twiecki** [comment]
hm, do we need `chain` at all?

**Comment by ColCarroll** [comment]
If i was guessing, I'd say it would be useful for generating multiple chains without multiprocessing, or maybe for to text backends and running things for multiple days.  

Just looking at the code and not having tried it, it looks somewhat brittle, particularly to `IndexError` if you're not being careful.  I'd support having less fine-grained control for generating chains -- if the user specifies `draws=1000, nchains=15, njobs=4`, then we should be able to work out how to do that.

**Comment by dreamqin68** [comment]
@twiecki @ColCarroll  Thanks for your answer. Ah, so I know that the njobs==nchains, and the 'chain' just a mark, I have realized that I have misunderstood the meaning of the "chain" parameter. I would appreciate an implementation that separates control of nchains and njobs.
